CHANGELOG.asciidoc

```diff
@@ -143,6 +143,7 @@ TinkerPop 3.2.6 (Release Date: NOT OFFICIALLY RELEASED YET)
 
 This release also includes changes from <<release-3-1-8, 3.1.8>>.
 
+* Fixed a bug in `RangeByIsCountStrategy`.
```

Comment:
Can you be a bit more explicit? -- e.g. "Fixed a bug regarding `ConnectiveSteps` in `RangeByIsCountStrategy`.